app.js

```diff
@@ -0,0 +1,6 @@
+function onoff() {
+  let element = document.getElementById("getid");

```

Comment:
Please don't use ID for elements and let's change name to more understandable according action.